notebooks/2023-01-23 Differential abundance testing.Rmd

```diff
@@ -23,14 +18,14 @@ library(tidyr)
 library(ggplot2)
 library(broom)
 library(biobroom)
-
-
+library(here)
+library(ggbeeswarm)
 ```
 
 ```{r}
 
 
-prot_res <- readRDS("/Users/user123/Documents/GitHub/OOPS-infSILAC-HepG2/notebooks/prot_res.rds")
+prot_res <- readRDS(here('results/prot_res.rds'))
```

Comment:
Using `here` will make the code work across different filesystems as it will define the path relative to the top of the R project, which is the same as the top of the repository.